subprojects/docs/src/docs/userguide/getting-started/tutorial/part4_gradle_plugins.adoc

```diff
@@ -173,7 +174,7 @@ BUILD SUCCESSFUL in 331ms
 
 The `publishToMavenLocal` task builds the POM file and the artifacts to be published. It then _installs_ them into the local Maven repository.
 
-You can view the POM and JAR file in the `build` directory:
+You can view the POM and JSON file in the `build` directory:
```

Comment:
Generally we call module metadata files `GMM`. It is technically in JSON format, but calling module metadata files `JSON` here is like calling POM files `XML`

Reply:
Thank you @jvandort!